CHANGES.md

```diff
@@ -13,6 +13,11 @@
 
 ### Fixed
 
+- Only show an error that there are no packages that build with dune when there
+  is no solution. The code would otherwise report an that no packages build
+  with dune if the selected package builds with dune, leading to confusion.
+  (#245, @Leonidas-from-XIV)
```

Comment:
What would you think of this phrasing instead:
```suggestion
- Fix a bug where a package which had a single version that built with dune and got selected by the solver
  would be reported has having no version building with dune. (#245, @Leonidas-from-XIV)
```
I think it make it a bit clearer what the bug was.